zfmt/ztests/decls.yaml

```
@@ -10,7 +10,9 @@ inputs:
       const foo = "bar"
       func add(a,b): (a+b)
       const bar = "baz"
-      pass
+      op stamp(outer, assignee): ( yield {...outer, assignee, ts: now()} )
+      op addbar(...): ( pass )
```

Nit: This doesn't add bar.
```suggestion
      op nop(...): ( pass )
```